addon/components/au-dropdown.js

```diff
@@ -60,6 +61,15 @@ export default class AuDropdown extends Component {
 
       return this.args.dropdownButtonLabel;
     } else if (this.args.buttonLabel) {
+      deprecate('@dropdownLabel is deprecated', false, {
+        id: '@appuniversum/ember-appuniversum.au-dropdown.dropdownButtonLabel-argument',
+        until: '2.0.0',
```

Comment:
Do we still want to target v2 since that's almost ready to go? Maybe we should target v3 instead so consumers have more time to remove the `dropdownLabel` usage?

I'm not sure how widely used it is so maybe it's not needed to keep it around for another major version.

Reply:
Also, it's not very clear what action the user should take to resolve the deprecation. Should they simply remove the argument?